prime-router/src/main/resources/metadata/hl7_mapping/resources/Observation/OBX.yml

```diff
@@ -112,7 +112,7 @@ elements:
       hl7XCNField: '%{hl7OBXField}-25'
 
   - name: observation-aoe-identifier
-    condition: '%resource.meta.tag.code="AOE"'
+    condition: '%resource.identifier.type.coding.code = "81959-9"'
```

Comment:
Are we confident that this will cover all the AOEs we will get from SR?  I don't think the integration tests have a comprehensive list